### PR TITLE
more JDK deps, libxml2, perl-test-pod, perl-yaml-syck, libxslt, xmlto…

### PR DESCRIPTION
**Pull request by rawlingsj**
…, libxau, xtrans

Signed-off-by: James Rawlings <jrawlings@chainguard.dev>

### CODE REVIEW

File: libxml2.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: MIT
+
+secfixes:
+  2.10.3-r0:
+    - CVE-2022-40303
+    - CVE-2022-40304
```
Comment (rawlingsj) [inline libxml2.yaml:17]: Is this correct?  If we add a new package with a secfix for the same version we are publishing, should we include these here?  
Also I've not included secfixes for previous versions?

**Comment by rawlingsj** [comment]
Sorry for the larger review.  There was a few related to each other so figured it best to push together.  I can push smaller or larger groups if preferred.

**Comment by rawlingsj** [comment]
hmm CI just failed with an error I've not seen when testing
```
2022/10/20 21:27:23 melange (libxml2/x86_64): /usr/lib/gcc/x86_64-pc-linux-gnu/12.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /lib/../lib64/libz.a(crc32.o): relocation R_X86_64_32 against `.rodata' can not be used when making a shared object; recompile with -fPIC
2022/10/20 21:27:23 melange (libxml2/x86_64): /usr/lib/gcc/x86_64-pc-linux-gnu/12.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: failed to set dynamic section sizes: bad value
```

**Comment by rawlingsj** [comment]
specifically the error `recompile with -fPIC` 

I'm not sure if that's the result of something I've missed from this PR or if it's a valid problem?

**Comment by rawlingsj** [comment]
tried a change but it didn't help so reverted back to the original commit + issue

File: libxml2.yaml (hunk 1 above)
Comment (kaniini) [reply libxml2.yaml:17]: yes, that seems fine

**Comment by kaniini** [comment]
It's because libxml2 build system has decided it wants to staticly link zlib in.  I split out the zlib static library into its own package, to help give the libxml2 build system an attitude adjustment.

**Comment by kaniini** [comment]
GNOME appears to have recut the libxslt release since you wrote libxslt.yaml: 2022/10/20 23:01:02 melange (libxslt/x86_64): sha256sum: WARNING: 1 of 1 computed checksums did NOT match

Sigh.

**Comment by rawlingsj** [comment]
> It's because libxml2 build system has decided it wants to staticly link zlib in. I split out the zlib static library into its own package, to help give the libxml2 build system an attitude adjustment.

thank you!

**Pull request merged**